src/components/TodoCreationForm.js

```diff
@@ -1,27 +1,52 @@
 import React from "react";
+import DatePicker from "react-datepicker";
+import Button from "./library/Button";
 
 export default class TodoCreationForm extends React.Component {
   state = {
-    taskName: ""
+    todoName: "",
+    dueDate: null
   }
 
   render() {
+    const { onCreate } = this.props
+    const { todoName, dueDate } = this.state;
+    const disabled = !todoName || !dueDate;
```

Comment:
I think it makes more sense to do things similarly to what we did in the `EditionForm`, that is: `canSubmit = !!todoName && !!dueDate`.
Reading `const disabled = ...` does not make understanding easier. We do not understand "disabled" refers to. But it is clear that `canSubmit` refers to a form, which directly rings a bell to the reader when in the context of a file called `TodoCreationForm`

Remember that adding `!!` before a variable is equivalent to converting the variable to its boolean counterpart.

Example:
```javascript
const a = 0; // 0 is considered "false" by javascript so ...
!a // => converts a to boolean, then take the inverse. In the end, evaluates to true
!!a // => converts a to boolean, reverse two times the boolean value so finally evaluates to false

const a = 1; // 1 is considered "false" by javascript (as well as all numerical value different from 0) so ...
!a // => converts a to boolean, then take the inverse. In the end, evaluates to false
!!a // => converts a to boolean, reverse two times the boolean value so finally evaluates to true

const a = "hello"; // All strings except the empty string "" is truthy in javascript so ...
!a // => converts a to boolean, then take the inverse. In the end, evaluates to true
!!a // => converts a to boolean, reverse two times the boolean value so finally evaluates to false
```

Reply:
lmao you did not do that for "canSubmit" BUT OKAY